Classes/Utility/AlertAction.swift

```diff
@@ -143,8 +143,18 @@ struct AlertAction {
     }
 
     static func bookmark(_ bookmark: BookmarkModel) -> UIAlertAction {
-        return UIAlertAction(title: NSLocalizedString("Add To Bookmark", comment: ""), style: .default) { _ in
-            BookmarksStore.shared.add(bookmark: bookmark)
+        let isNewBookmark = !BookmarksStore.shared.contains(bookmark: bookmark)
+        let title = isNewBookmark ? Constants.Strings.bookmark : Constants.Strings.removeBookmark
+        return UIAlertAction(
+            title: title,
+            style: isNewBookmark ? .default : .destructive
```

Comment:
Actually can we just make this `.default`? Seeing the destructive "red" next to other actions like "Close" and "Lock" is a little weird b/c this doesn't carry _nearly_ the same weight. cc @BasThomas for thoughts

Reply:
I think it would make sense, but I think we’d like to move the action away from lock/close. Also, both Bookmark and Remove Bookmark have “side effects”, so (as we’re using it right now) both actions should be “destructive”. 

I feel like it’s time to explore something completely different than the (now very long) action sheets...

Reply:
"Remove bookmark" does not carry the same weight as other destructive buttons. But IMO it would avoid accidentally tapping. 

From Apple doc
> case destructive
>Apply a style that indicates the action might change or delete data.

Also, I do agree with @BasThomas. Action sheet is very long now. After 1.12.0 we should rethink and adopt a better way to handle this action sheet for sure

Reply:
What do you think about moving the action (more) to the bottom? I think it doesn't make sense that it is next (and in this case even above) some probably more used, destructive actions.

Reply:
On the discussion regarding "destructive": https://github.com/rnystrom/GitHawk/pull/532

Reply:
You mean "Remove bookmark" action to the bottom of the action sheet? 
Yes, I'm down with that.

Reply:
Yep, the add + remove bookmark :)

Reply:
Moved bookmark actions to bottom of the action sheet. 👍